packages/askui-nodejs/src/core/model/annotation-result/detected-element.ts

```diff
@@ -13,8 +11,6 @@ export class DetectedElement {
   static fromJson(detectedElement: DetectedElement, resizeRatio = 1) {
     return new DetectedElement(
       detectedElement.name,
-      detectedElement.truncated,
-      detectedElement.difficult,
```

Comment:
did you test if annotate or interactive annotation is working?

Reply:
yes, tested both